index.js

```diff
@@ -62,6 +70,10 @@ const CLOSINGSTATES = [STATE.DOOR_OPENED_CLOSE]
 // These states will start a Brave Alert session for a location
 const ALERTSTARTSTATES = [STATE.SUSPECTED_OD]
 
+// Sentry monitoring Middleware
+app.use(Sentry.Handlers.requestHandler())
+app.use(Sentry.Handlers.tracingHandler())
+
```

Comment:
What do you think about putting this also into a `sentryHelper.setup()` function in Brave Alert Lib?